.github/workflows/app.yml

```diff
@@ -5,7 +5,7 @@ name: Python application
 
 on:
   push:
-    branches: [ master, rest ]
+    branches: [ master, swagger ]
```

Comment:
Its not important, you can not change this rule, better to put only master in your case